dds_web/commands.py

```diff
@@ -345,6 +345,397 @@ def lost_files_s3_db(action_type: str):
         flask.current_app.logger.info("Found no lost files")
 
 
+@click.command("set-available-to-expired")
+@flask.cli.with_appcontext
+def set_available_to_expired():
```

Comment:
Add docstring with information on how often it runs and when. Get from apscheduler.